validate.py

```diff
@@ -192,6 +192,54 @@ def validate_card(args, card, card_schema, pack_code, factions_data, types_data,
         validation_errors += 1
         verbose_print(args, "%s\n" % e.message, 0)
 
+def verify_stripped_text_is_ascii(args, card, pack_code):
+    global validation_errors
+
+    stripped_text = card.get('stripped_text', '')
+    try:
+        stripped_text.encode('ascii')
+    except UnicodeEncodeError:
+        verbose_print(args, "ERROR\n", 2)
+        verbose_print(
+            args,
+            "Stripped text contains non-ascii characters in card: (pack code: '{}' title: '{}' stripped_text '{}')\n".format(
+                pack_code,
+                card['title'],
+                stripped_text,
+            ),
+            0,
+        )
+        validation_errors += 1
+
+def verify_text_has_fancy_text(args, card, pack_code):
+    global validation_errors
+
+    text = card.get('text', '')
+    if ('[interrupt]' in text) and ('[interrupt] – ' not in text):
+        verbose_print(args, "ERROR\n", 2)
+        verbose_print(
+            args,
+            "Incorrect interrupt text in card: (pack code: '{}' title: '{}' text '{}')\n".format(
+                pack_code,
+                card['title'],
+                text,
+            ),
+            0,
+        )
+        validation_errors += 1
+    if ('Interface' in text) and (('Interface ->' in text) or ('Interface →' not in text)):
```

Comment:
why accept both formats instead of just the one?

Reply:
This is checking the "text" field, so it should be "Interface →" (with the unicode arrow). I'm raising an error if it's a dash and greater than or anything that's not a unicode error.